pkg/proto/block.go

```diff
@@ -321,15 +321,16 @@ type BlockHeader struct {
 	RewardVote             int64        `json:"desiredReward"`
 	ConsensusBlockLength   uint32       `json:"-"`
 	NxtConsensus           `json:"nxt-consensus"`
-	TransactionBlockLength uint32            `json:"transactionBlockLength,omitempty"`
-	TransactionCount       int               `json:"transactionCount"`
-	GeneratorPublicKey     crypto.PublicKey  `json:"generatorPublicKey"`
-	BlockSignature         crypto.Signature  `json:"signature"`
-	TransactionsRoot       B58Bytes          `json:"transactionsRoot,omitempty"`
-	StateHash              *crypto.Digest    `json:"stateHash,omitempty"`        // is nil before protocol version 1.5
-	ChallengedHeader       *ChallengedHeader `json:"challengedHeader,omitempty"` // is nil before protocol version 1.5
-
-	ID BlockID `json:"id"` // this field must be generated and set after Block unmarshalling
+	TransactionBlockLength uint32              `json:"transactionBlockLength,omitempty"`
+	TransactionCount       int                 `json:"transactionCount"`
+	GeneratorPublicKey     crypto.PublicKey    `json:"generatorPublicKey"`
+	BlockSignature         crypto.Signature    `json:"signature"`
+	TransactionsRoot       B58Bytes            `json:"transactionsRoot,omitempty"`
+	StateHash              *crypto.Digest      `json:"stateHash,omitempty"`        // is nil before protocol version 1.5
+	ChallengedHeader       *ChallengedHeader   `json:"challengedHeader,omitempty"` // is nil before protocol version 1.5
+	FinalizationVoting     *FinalizationVoting `json:"finalizationVoting,omitempty"`
```

Comment:
Guess need comment when this field can be `nil` and when cannot.
Also, please, add safe getter for this field (I mean like `GetChallengedHeader` and `GetStateHash`)